### ISSUE TITLE
Like , send and reply buttons are inactive

### ISSUE THREAD

**Issue by MortezaIsvand** [adrianhajdin/threads#43]
these three buttons not work. Home page is not responsive. It's kind of ugly

**Comment by yuvraj042003** [comment]
Yes Same error in my project. I have my reply button is not working.

**Comment by yuvraj042003** [comment]
If you get ant idea related to this then plz inform me.

**Comment by baxsm** [comment]
These three buttons are not implemented at all and I don't think so it'll be implemented at all in this current repo.
And other errors are related to mongodb so make sure your connection string is valid

**Comment by yuvraj042003** [comment]
Yes They not implemented these buttons. So I think we need to implement itself.

**Comment by VictorAndrade11** [comment]
when someone implements the functionality of the like, share and send buttons please send me!

**Comment by magh20** [comment]
> these three buttons not work. Home page is not responsive. It's kind of ugly

Hello, Are you iranian?
If yes, Then how did you solve the VPN problem?
Error: querySrv ENOTFOUND _mongodb._tcp.cluster.3caigv3.mongodb.net
I got this error and i think its because VPN, Do you see this error?

**Comment by MortezaIsvand** [comment]
Hello, you have to apply some changes in your mongo db account. Go to your
account, network access, edit IP access and allow access from anywhere (
0.0.0.0/1). this will make our database accessible from all IP addresses

در تاریخ یکشنبه ۳ سپتامبر ۲۰۲۳،‏ ۲:۵۳ mohammad amin ghorbani <
***@***.***> نوشت:

**Comment by yuvraj042003** [comment]
> > these three buttons not work. Home page is not responsive. It's kind of ugly
> 
> Hello, Are you iranian? If yes, Then how did you solve the VPN problem? Error: querySrv ENOTFOUND _mongodb._tcp.cluster.3caigv3.mongodb.net I got this error and i think its because VPN, Do you see this error?



> Hello, you have to apply some changes in your mongo db account. Go to your account, network access, edit IP access and allow access from anywhere ( 0.0.0.0/1). this will make our database accessible from all IP addresses.

Yes By follow these instruction your project working well.
